app/ui/content/index.js

```diff
@@ -0,0 +1,23 @@
+/*
+ Copyright 2016 Mozilla
+
+ Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ this file except in compliance with the License. You may obtain a copy of the
+ License at http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software distributed
+ under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ specific language governing permissions and limitations under the License.
+ */
+
+import { bindHistorySearch, fetchRecentHistory } from './history';
+import { fetchRecentBookmarks } from './stars';
+
+if (~document.location.pathname.indexOf('history')) {
+  bindHistorySearch();
+  fetchRecentHistory();
+}
+
+if (~document.location.pathname.indexOf('stars')) {
+  fetchRecentBookmarks();
+}
```

Comment:
Even aside from these tests being susceptible to accidental breakage (if we add a "recent_history.html" for example) it doesn't seem right to have to have a single entry point for every content page we add.